### PR TITLE
kb(grid,tooltip):tooltip in grid example changes

### PR DESCRIPTION
**Pull request by xristianstefanov**
- Added a new page with an example that shows only one instance of the Tooltip for the whole Grid.

### CODE REVIEW

**Comment by marin-bratanov** [comment]
why was "groupable" removed? Does this work as-is without any calls for the tooltip to re-evaluate its target? Is it "automatic" or through explicit API? There should also be an update on the readme to explain the two pages and how each works.

**Comment by xristianstefanov** [comment]
> why was "groupable" removed? Does this work as-is without any calls for the tooltip to re-evaluate its target? Is it "automatic" or through explicit API? There should also be an update on the readme to explain the two pages and how each works.

Hi Marin! The "groupable" was unintentionally removed. The tooltip re-evaluates its targets "automatically". I also fixed the readme.

File: tooltip/in-grid/Pages/Index.razor [hunk 1]
```diff
@@ -17,20 +17,21 @@
...
                 @{
                     BasicEmployee employee = context as BasicEmployee;
-                    <span data-employeeId="@employee.Id" id="@( "tooltip-target" + employee.Id)">@employee.Name</span>
-                    <TelerikTooltip TargetSelector="@( "#tooltip-target" +  employee.Id)"
-                                    Width="350px" Height="250px" Position="@TooltipPosition.Right">
-                        <Template Context="ttipContext">
-                            <EmployeeDetails TargetData="@ttipContext.DataAttributes" />
-                        </Template>
-                    </TelerikTooltip>
+                    <span data-employeeId="@employee.Id" id="@( "tooltip-target")">@employee.Name</span>
```
Comment (dimodi) [inline tooltip/in-grid/Pages/Index.razor:20]:
```suggestion
                    <span data-employeeId="@employee.Id" id="tooltip-target">@employee.Name</span>
```

File: tooltip/in-grid/Pages/Index.razor [hunk 2]
```diff
@@ -17,20 +17,21 @@
...
                 }
             </Template>
         </GridColumn>
         <GridColumn Field="@(nameof(BasicEmployee.Team))" Title="Team" />
     </GridColumns>
 </TelerikGrid>
 
+<TelerikTooltip TargetSelector="#tooltip-target"
```
Comment (dimodi) [inline tooltip/in-grid/Pages/Index.razor:28]:
```suggestion
<TelerikTooltip TargetSelector=".tooltip-target"
```

**Review by dimodi — CHANGES_REQUESTED** [review]
The single Tooltip example cannot rely on an ID selector.

**Review by dimodi — APPROVED** [review]

**Pull request merged**